src/test/scala/specs/utils/DataGen.scala

```diff
@@ -1,28 +1,27 @@
 package specs.utils
 
-import com.danielasfregola.randomdatagenerator.RandomDataGenerator
-import org.scalacheck.{Arbitrary, Gen}
-
 import java.util.UUID
 
 case class Person(id: UUID, name: String, surname: String, dob: String, balance: BigDecimal)
 
-trait DataGen extends RandomDataGenerator {
-  implicit val arbitraryPerson: Arbitrary[Person] = Arbitrary {
-    for {
-      id <- Gen.uuid
-      name <- Gen.alphaStr
-      surname <- Gen.alphaStr
-      dob <- Gen.calendar
-      balance <- Arbitrary.arbBigDecimal.arbitrary
-    } yield {
+object DataGen {
+  private val text: String =
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et " +
+      "dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex " +
+      "ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat " +
+      "nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit " +
+      "anim id est laborum."
+
+  def gen(size: Int): List[Person] = {
+    List.fill(size)(
       Person(
-        id,
-        name,
-        surname,
-        dob.getTime.toString,
-        balance
+        UUID.fromString("90d08192-c32e-450b-b3c3-a3ec7d87bf83"),
+        "Lorem ipsum",
```

Comment:
Let's use a larger string here if we can so we can see how the cells behave pls

Reply:
Got it but still, let's increase it just to observe how it will behave in such cases (i.e. line break etc.)